cmd/launcher/desktop.go

```diff
@@ -83,6 +88,18 @@ func runDesktop(args []string) error {
 		)
 	}
 
+	if *flPpid <= 1 {
+		ppid, err := parentProcessId()
```

Comment:
I think we should log here if the `flPpid` was not provided

Reply:
Agreed! Added

Reply:
Sample logs after update if PPID flag is omitted:

```
{"caller":"runner.go:647","component":"desktop_users_processes_runner","level":"info","msg":"{\"caller\":\"desktop.go:92\",\"fl_ppid\":0,\"msg\":\"expected PPID command-line flag for launcher desktop but did not receive it\",\"pid\":10985,\"severity\":\"warn\",\"subprocess\":\"desktop\",\"ts\":\"2023-02-22T20:43:27.218669Z\",\"uid\":\"501\"}","subprocess":"desktop","ts":"2023-02-22T20:43:27.218759Z","uid":"501"}
{"caller":"runner.go:647","component":"desktop_users_processes_runner","level":"info","msg":"{\"caller\":\"logutil.go:13\",\"err\":\"run with positional args: running subcommand desktop: valid ppid not included in flags and could not be looked up: could not find parent process for sudo launcher desktop: could not find parent line\",\"severity\":\"info\",\"ts\":\"2023-02-22T20:43:27.244036Z\"}","subprocess":"desktop","ts":"2023-02-22T20:43:27.244158Z","uid":"501"}
```

Reply:
After latest commit:

```
{"caller":"runner.go:647","component":"desktop_users_processes_runner","level":"info","msg":"{\"caller\":\"logutil.go:13\",\"err\":\"run with positional args: running subcommand desktop: received invalid PPID command-line flag for launcher desktop: 0\",\"severity\":\"info\",\"ts\":\"2023-02-22T20:52:08.481721Z\"}","subprocess":"desktop","ts":"2023-02-22T20:52:08.481866Z","uid":"501"}
```